clang-tools-extra/include-cleaner/unittests/WalkASTTest.cpp

```diff
@@ -534,6 +534,8 @@ TEST(WalkAST, Enums) {
   testWalk(R"(namespace ns { enum E { A = 42 }; }
               struct S { using ns::E::A; };)",
            "int e = S::^A;");
+  testWalk(R"(namespace ns { enum E { $explicit^A = 42 }; })",
+           "namespace z = ns; int e = z::^A;");
```

Comment:
Could you add a test for the global specifier too?
```::A``` has a slightly different representation as `NestedNameSpecifier`, so we might want to make sure this case is also tested. (I **think** the existing code should work just fine, though).

Reply:
ugh, apparently for global namespaces, we don't store any decls. hence we were failing. tweaked the code a little.

Reply:
so it was a good test case! thanks, LG!